pkg/d06de9dc-d4ab-4de4-ac44-b544e4ff80bb/package.lp

```diff
@@ -0,0 +1,204 @@
+(librepcb_package d06de9dc-d4ab-4de4-ac44-b544e4ff80bb
+ (name "Sovtek IN-12")
```

Comment:
Package naming convention:
```suggestion
 (name "SOVTEK_IN-12")
```

Reply:
I had it like that initially, but then I re-read the package naming convention 😄

![screenshot-20241224-230546](https://github.com/user-attachments/assets/8aab843e-cd0c-4e19-8859-c085307623ba)

We should probably adjust that, or maybe add a dedicated item about manufacturer-specific naming.

Reply:
Huh, I was not aware the conventions are outdated :scream:

Fixed in https://github.com/LibrePCB/librepcb-doc/pull/78.